gate-web/gate-web.gradle

```diff
@@ -35,7 +35,7 @@ dependencies {
   compile("com.google.api-client:google-api-client:1.21.0")
   compile("com.google.apis:google-api-services-admin-directory:directory_v1-rev65-1.21.0")
   compile("org.springframework.security.extensions:spring-security-saml2-core:1.0.2.RELEASE")
-  compile("org.springframework.security.extensions:spring-security-saml-dsl:1.0.0.M1") {
+  compile("org.springframework.security.extensions:spring-security-saml-dsl:1.0.0.M2") {
```

Comment:
They are cutting milestones because you are pushing changes to this DSL, correct? Do we have an idea when this will be in a stable release?